operator-common/src/main/java/io/strimzi/operator/common/model/Labels.java

```diff
@@ -97,7 +97,7 @@
      * Used to exclude parent CR's labels from being assigned to provisioned subresources
      */
     public static final Pattern STRIMZI_LABELS_EXCLUSION_PATTERN = Pattern.compile(System.getenv()
-            .getOrDefault("STRIMZI_LABELS_EXCLUSION_PATTERN", "^app.kubernetes.io/(?!part-of).*"));
+            .getOrDefault("STRIMZI_LABELS_EXCLUSION_PATTERN", "(^app.kubernetes.io/(?!part-of).*|^kustomize.toolkit.fluxcd.io.*)"));
```

Comment:
why are we setting the kustomize exclusion by default and not letting the user to do so using the `STRIMZI_LABELS_EXCLUSION_PATTERN` env var?

Reply:
So, you prefer to:
* Let the user run into problems
* Search for a solution
* Open an issue / discussion
* Wait for Jakub to help them?

Reply:
I am just saying that this list could grow with any kind of other projects labels if we start to add the first one.

Reply:
I agree, but I'm not sure I see anything wrong with it. At the end that was one of the purposes with which we created it, or?